### DISCUSSION TITLE
Report | Week 14

### DISCUSSION THREAD

**Discussion by LeonmAndreesen** [SE-TINF22B6/time2shine#82] [Show and tell]
In todays blog we like to show you the things we have dealt with. We are still working on the website. We have come to the conclusion that there will be some new games. First of all we have worked on the database. Besides that we have put time into finishing some layout on the website and doing the mandatory tests

**Comment by sfrancisw** [comment]
edit: we also created our test plan and added it to our documentation. Right now, we are mostly working on our JUnit tests in the backend to make sure that everything is working as intended. The API integration for the blackjack cards is now also working and we plan to implement it to our main branch in the upcoming week. 

Our snake game also made some progress. We'll need some actual snake graphics though to show you the process, as this is how it currently looks with temporary graphics:
![grafik](https://github.com/SE-TINF22B6/time2shine/assets/49996344/31844eb6-0442-4ad3-a97a-83d9e7c467d8)

**Comment by Nahlam4** [comment]
Dear Team time2shine, I see you are progressing nicely. My only question would be wich games you have decided you want to implement.
Im exited to see your games come alive and, if you choose to use beta tests, i would be willing to test your games.

Keep going in the next sprint and keep progressing.

-Your DHBWhub Team

**Comment by nuo-li** [comment]
Dear Time2Shine, you mentioned "We plan to test different aspects of our back- and frontend with JUnit tests" in your documentation.  How do you test frontend with JUnit?
